client/jetpack-cloud/sections/partner-portal/payment-method-actions/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import EllipsisMenu from 'calypso/components/ellipsis-menu';
+import PopoverMenuItem from 'calypso/components/popover-menu/item';
+import { PaymentMethod } from 'calypso/lib/checkout/payment-methods';
+
+import './style.scss';
+
+interface Props {
+	card: PaymentMethod;
```

Comment:
This interface will be used in [the follow-up PR.](https://github.com/Automattic/wp-calypso/pull/61247)